packages/amplify-graphql-rds-model-transformer/rds-lambda/clients/DBClient.ts

```diff
@@ -0,0 +1,6 @@
+import { Knex } from 'knex';
+
+export interface DBClient {
+    client: Knex;
+    getClient(): Promise<any | Knex<any, any[]>>;
```

Comment:
Is including `any` here meant to be future proof for using non-Knex clients?

Reply:
the any is just an artifact of knex - it's primarily a JS library with not fully fleshed out support for typing